client/me/purchases/manage-purchase/style.scss

```diff
@@ -313,7 +313,3 @@
 		padding: 16px 24px;
 	}
 }
-
-.manage-purchase__detail-date-span {
-	white-space: nowrap;
-}
```

Comment:
I'm not sure why this is being removed -- to me it looked much better before.

Before:

![before](https://user-images.githubusercontent.com/235183/104878288-a31c3c80-5929-11eb-83c7-ea8df04317d9.png)

After:

![after](https://user-images.githubusercontent.com/235183/104878286-a31c3c80-5929-11eb-90ef-c4c185eaf2b3.png)

Reply:
But have you tried it at smaller widths?

<img width="269" alt="Screen Shot 2021-01-18 at 11 54 21 AM" src="https://user-images.githubusercontent.com/2036909/104943568-ee633900-5983-11eb-8209-678fe0c82e10.png">